templates/project/.github/workflows/test.yaml.twig

```diff
@@ -68,10 +68,11 @@ jobs:
               with:
                   {% verbatim %}php-version: ${{ matrix.php-version }}{% endverbatim %}
                   coverage: pcov
-{% if branch.hasService('mongodb') %}
```

Comment:
hmm it looks ok, but I think this hasService('mongo') is used somewhere 🤔

Reply:
What do you mean ?

Reply:
did you search the whole project for "hasService('mongodb')"? Makefiles etc?